druntime/src/core/sync/condition.d

```diff
@@ -84,7 +84,8 @@ class Condition
     /// ditto
     this( shared Mutex m ) shared nothrow @safe @nogc
     {
-        this(m, true);
+        import core.atomic : atomicLoad;
+        this(atomicLoad(m), true);
```

Comment:
This one is a bit silly, the address of `m` can never be shared at this point. I wonder if there could be a way to express that `shared Mutex m` only applies to indirections of `m`, similar to `const(int)*` instead of `const int*`.

Reply:
I agree - but I didn't know what else to do.